.github/workflows/build.yml

```diff
@@ -32,14 +36,20 @@ jobs:
         architecture: 'x86'
     - name: Set up Python 3.8 x64
       uses: actions/setup-python@v2
-      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
+      if: ${{ matrix.os == 'ubuntu-latest' }}
```

Comment:
Braces shouldn't be needed here. The conditional is already an expression.

Reply:
fixed.